tests/run

```diff
@@ -0,0 +1,2 @@
+#! /usr/bin/env bash
+python -m unittest discover --start-directory tests
```

Comment:
`python setup.py test` should suffice - I'm not sure we need this script unless we're also linting, which I don't think is in the scope of #1, but rather #14